src/py42/sdk/queries/__init__.py

```diff
@@ -8,6 +8,7 @@ def __init__(self, *args, **kwargs):
         self._group_clause = kwargs.get(u"group_clause", u"AND")
         self.page_number = kwargs.get(u"page_number") or 1
         self.page_size = kwargs.get(u"page_size") or settings.security_events_per_page
+        self.pgToken = kwargs.get(u"pg_token") or None
```

Comment:
This should be snake_case, not camelCase

Reply:
changed.